src/index.css

```
@@ -1,50 +1,1071 @@
+.nav-card-value,
+.budget-label {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  max-width: 100%;
+}
+
+.transaction p,
+.transaction-info p {
+  overflow-wrap: break-word;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)